src/expander/expander.c

```diff
@@ -55,7 +55,7 @@ void	expander_loop_dollar(t_simple_cmds *content, int i, t_data *data)
 		if (content->cmd[i][j] == '$')
 		{
 			dollar_flag = checker_dollar(content->cmd[i], j);
-			delete_quotes(content->cmd, i);
+			content->cmd[i] = delete_quotes(content->cmd[i]);
```

Comment:
Making delete_quotes universal, so I can use it to fix the redirects with quotes problem :)